runtime/compiler/runtime/Listener.cpp

```diff
@@ -348,9 +348,38 @@ static int32_t J9THREAD_PROC listenerThreadProc(void * entryarg)
       {
       OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
       char timestamp[32];
+      char zoneName[32];
+      int32_t zoneSecondsEast = 0;
+      TR_VerboseLog::CriticalSection vlogLock;
 
       omrstr_ftime_ex(timestamp, sizeof(timestamp), "%b %d %H:%M:%S %Y", j9time_current_time_millis(), OMRSTR_FTIME_FLAG_LOCAL);
-      TR_VerboseLog::writeLineLocked(TR_Vlog_INFO, "StartTime: %s", timestamp);
+      TR_VerboseLog::writeLine(TR_Vlog_INFO, "StartTime: %s", timestamp);
+
+      TR_VerboseLog::write(TR_Vlog_INFO, "TimeZone: ");
+      if (-1 == omrstr_current_time_zone(&zoneSecondsEast, zoneName, sizeof(zoneName)))
```

Comment:
I think it's preferable to check for zero. In the future, the function may distinguish different failures with different non-zero return values. Same applies to the use in `javadump.cpp`.
```
if (0 != omrstr_current_time_zone(&zoneSecondsEast, zoneName, sizeof(zoneName)))
```

Reply:
JIT coding convention requires the `if` part to be enclosed in brackets, if the `else` part has brackets.